### DISCUSSION TITLE
"utoipa_auto_discovery" crate (auto detection of Paths/Schemas)

### DISCUSSION THREAD

**Discussion by rxdiscovery** [juhaku/utoipa#635] [Show and tell]
Hello,

(this is my small contribution to this wonderful community)

Here's a first version of a crate (utoipa_auto_discovery) that automates Paths and Schemas detection on Utoipa.
I'm currently using it on an internal project,

Url of the crate :
https://crates.io/crates/utoipa_auto_discovery

it's not perfect yet, but it works well :+1:  , you can try it and give me your feedback.

Thanks..

**Comment by juhaku** [comment]
Awesome, I'll try to check it out sometime soon.

**Comment by jayvdb** [comment]
Note it only does path detection, not schema detection yet, but that is in progress.
When it does do schemas, it has solved https://github.com/juhaku/utoipa/issues/465

**Reply by rxdiscovery** [reply]
I'm trying to find an ergonomic way to detect schemas, 
in a week I'll post a new version including the missing functionality.
